setup.py

```diff
@@ -30,7 +25,7 @@ def read_version():
     return finder.version
 
 
-dependencies = filter(bool, map(bytes.strip, local_file('requirements.txt').splitlines()))
+dependencies = [x.split('==')[0] for x in open('requirements.txt').readlines()]
```

Comment:
- Please do not use the methods `file.readline()` and `file.readlines()` to iterate over the file/stream:

```python
# Bad
while True:
    line = your_file.readline()
    ...

for line in your_file.readlines():
    ...


# Better
for line in your_file:
    ...
```